web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -44,14 +45,17 @@ export const SidebarCardsList = (props: Props): ReactElement => {
   };
 
   const hiddenCardsAccordion = (): ReactNode => {
-    if (props.displayCertifications) {
+    if (props.displayCertifications || props.displayFundings) {
```

Comment:
[Pebble] does this need a test, it's new logic?

Reply:
I added the test

Reply:
I was wondering if there are any tests regarding what's being returned based on this conditional that need to be updated or newly added